app/src/main/java/com/zeapo/pwdstore/utils/PreferenceKeys.kt

```diff
@@ -0,0 +1,43 @@
+package com.zeapo.pwdstore.utils
+
+object PreferenceKeys {
+    const val GIT_SERVER_INFO = "git_server_info"
+    const val SSH_OPENKEYSTORE_CLEAR_KEY_ID = "ssh_openkeystore_clear_keyid"
+    const val GIT_CONFIG = "git_config"
+    const val SSH_KEY = "ssh_key"
+    const val SSH_KEYGEN = "ssh_keygen"
+    const val CLEAR_SAVED_PASS = "clear_saved_pass"
+    const val SSH_SEE_KEY = "ssh_see_key"
+    const val GIT_DELETE_REPO = "git_delete_repo"
+    const val GIT_EXTERNAL = "git_external"
+    const val PREF_SELECT_EXTERNAL = "pref_select_external"
+    const val OPENPGP_KEY_ID_PREF = "openpgp_key_id_pref"
+    const val GENERAL_SHOW_TIME = "general_show_time"
+    const val CLEAR_CLIPBOARD_20X = "clear_clipboard_20x"
+    const val AUTOFILL_ENABLE = "autofill_enable"
+    const val OREO_AUTOFILL_DIRECTORY_STRUCTURE = "oreo_autofill_directory_structure"
+    const val OREO_AUTOFILL_DEFAULT_USERNAME = "oreo_autofill_default_username"
+    const val OREO_AUTOFILL_CUSTOM_PUBLIC_SUFFIXES = "oreo_autofill_custom_public_suffixes"
+    const val AUTOFILL_APPS = "autofill_apps"
+    const val AUTOFILL_DEFAULT = "autofill_default"
+    const val AUTOFILL_ALWAYS = "autofill_always"
+    const val AUTOFILL_FULL_PATH = "autofill_full_path"
+    const val GIT_EXTERNAL_REPO = "git_external_repo"
+    const val USE_GENERATED_KEY = "use_generated_key"
+    const val APP_VERSION = "app_version"
+    const val SSH_OPENKEYSTORE_ID = "ssh_openkeystore_keyid"
+    const val OPENPGP_LIST_PROVIDER = "openpgp_provider_list"
+    const val HTTPS_PASSWORD = "https_password"
+    const val SSH_KEY_LOCAL_PASSPHRASE = "ssh_key_local_passphrase"
+    const val SSH_OPENKEYSTORE_KEYID = "ssh_openkeystore_keyid"
+    const val REPOSITORY_INITIALIZED = "repository_initialized"
+    const val REPO_CHANGED = "repo_changed"
+    const val EXPORT_PASSWORDS = "export_passwords"
+    const val ENABLE_DEBUG_LOGGING = "enable_debug_logging"
+    const val BIOMETRIC_AUTH = "biometric_auth"
+    const val PREF_KEY_CUSTOM_DICT = "pref_key_custom_dict"
+    const val PREF_KEY_IS_CUSTOM_DICT = "pref_key_is_custom_dict"
+    const val PREF_KEY_PWGEN_TYPE = "pref_key_pwgen_type"
+    const val OPENPGP_KEY_IDS_SET = "openpgp_key_ids_set"
+
+}
```

Comment:
needs a trailing newline, run a format pass with Android Studio.

Reply:
Cool, need to update a lot more files though.